src/cron-jobs/index.ts

```diff
@@ -3,9 +3,10 @@ import cron from 'node-cron';
 import { completeAllActiveChallengesInTheEndOfMonth } from '~/cron-jobs/jobs/challenge-completion';
 
 cron.schedule(
-  '0 0 28-31 * *',
+  '*/5 22-23 28-31 * *',
```

Comment:
Looks like we will have a bug with end date of month. We don`t have validation on "How much days in the month?". And if the December has 31 days, cron will execute on the 28th day, because there in JS code no validation

Reply:
Sorry, I`ve double check the implementation and everything good